### PR TITLE
Added a helpful message for when JS is disabled

### LINKED ISSUE #5913: Warning when browser has Javascript disabled?
Not sure how often this comes up, but when Javascript is disabled, the site is very broken. 
screen shot 2015-09-10 at 11 14 40 pm

Which is to be expected, since it runs on AngularJS.

Facebook also needs Javascript to run properly, so when you have Javascript disabled and try to to go Facebook.com, the following screen appears:

screen shot 2015-09-10 at 11 15 18 pm

Was wondering if something like that might be useful for Habitica?
## <bountysource-plugin>

Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/26605254-warning-when-browser-has-javascript-disabled?utm_campaign=plugin&utm_content=tracker%2F68393&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F68393&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by gauthamchandra**
Fixes #5913 

The message now displays a link to [http://www.enable-javascript.com/](http://www.enable-javascript.com/) so that the user has browser specific instructions on how to re-enable JavaScript.

**How it looks on desktop when signed in**:
image

**How it looks on mobile when signed in**:
image

**How it looks on the landing page when not signed in**:
image

### CODE REVIEW

**Comment by gauthamchandra** [comment]
Fixed capitalization so that the text now reads `Please enable JavaScript to continue!` instead of `Please Enable JavaScript to continue!`

File: website/views/static/front.jade [hunk 1]
```diff
@@ -40,6 +40,14 @@ html(ng-app='habitrpg', ng-controller='RootCtrl')
     include ../shared/avatar/index
     include ../shared/mixins
     include ../shared/modals/members
+
+    noscript(class='banner')
```
Comment (crookedneighbor) [inline website/views/static/front.jade]: Why not `noscript.banner`?

File: website/views/static/front.jade [hunk 2]
```diff
@@ -40,6 +40,14 @@ html(ng-app='habitrpg', ng-controller='RootCtrl')
...
+      | Alas! Your browser doesn't have JavaScript enabled and without it,
```
Comment (crookedneighbor) [inline website/views/static/front.jade]: Shouldn't these be translatable strings?

**Comment by crookedneighbor** [comment]
Looking good :+1:

File: website/views/static/front.jade (hunk 1 above)
Comment (gauthamchandra) [inline website/views/static/front.jade]: Good point. Fixing. :+1:

File: website/views/static/front.jade (hunk 2 above)
Comment (gauthamchandra) [inline website/views/static/front.jade]: Good point. Will make them a translatable string.

**Comment by gauthamchandra** [comment]
@crookedneighbor updated PR. tell me if it's good to :ship: .

**Comment by crookedneighbor** [comment]
Looks good to me. I adjusted it to make the enable javascript link a button.

Tier awarded! You can read more about [contributor tiers and rewards here](http://habitrpg.wikia.com/wiki/Contributor_Rewards).

Note that as your tier increases, it takes progressively more work to achieve the next one, so for the higher tiers, larger PRs or a few PRs are required. There's some information about that at Contributor Rewards, but feel free to ask here if you have questions!

**Comment by gauthamchandra** [comment]
Cool. :+1: thanks!

**Pull request merged**